### DISCUSSION TITLE
[Question] Clarification to "Move existing wallpapers to new folder"

### DISCUSSION THREAD

**Discussion by 03302024** [rocksdanister/lively#2245] [General]
Under Settings >> General >> File, there is a check box that reads:
Move existing wallpapers to new folder

I was wondering what does it actually do, does it removes (deletes) files from their original locations, or it just copies them to a new location?

**Comment by rocksdanister** [comment]
Copies then delete (move.)

**Comment by 03302024** [comment]
Do you mean copies files from old library location to new library location, and then deletes files from old library location? Not deleting original files that are somewhre else on the disk, right?

**Reply by rocksdanister** [reply]
yes, lively only remembers file location for imported wallpapers (unless its lively .zip format then its extracted.)

**Reply by 03302024** [reply]
Very good, thank you. May be you want to change the wording to someting like this:
Move existing wallpaper library to new folder
